packages/babel/src/index.js

```diff
@@ -1,4 +1,5 @@
 import * as babel from '@babel/core';
+import { VERSION } from "rollup";
```

Comment:
I'm not 100% sure but it probably would be better to grab this from this: https://rollupjs.org/guide/en/#thismeta-rollupversion-string-watchmode-boolean . This would require moving `unpackInputPluginOptions` call to one of the hooks (`buildStart`?) though.

Reply:
Good catch. One more argument is that importing `rollup` from a plugin is not guaranteed to give you the correct Rollup version in more complex npm dependency scenarios, especially if Rollup is used programmatically while `this.meta.rollupVersion` is guaranteed to be correct.